proto/vendor/tendermint/rpc/grpc/types.proto

```diff
@@ -19,13 +19,17 @@ message RequestBroadcastTx {
 message ResponsePing {}
 
 message ResponseBroadcastTx {
-  tendermint.abci.ResponseCheckTx   check_tx   = 1;
-  tendermint.abci.ResponseDeliverTx deliver_tx = 2;
+  tendermint.abci.ResponseCheckTx check_tx  = 1;
+  tendermint.abci.ExecTxResult    tx_result = 2;
 }
 
 //----------------------------------------
 // Service Definition
 
+// BroadcastAPI
+//
+// Deprecated: This API will be superseded by a more comprehensive gRPC-based
+// broadcast API, and is scheduled for removal after v0.38.
```

Comment:
I think we're using _other_ broadcast, one from `proto/vendor/cosmos/tx/v1beta1/service.proto`, but please double check me.